<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventShow</class>
 <widget class="QWidget" name="EventShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="EventName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code SemiBold</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Event</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartTime">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>45</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>StartTime</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EndTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>45</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>EndTime</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DetailedInfomation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>45</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Detailed</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>12</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Address&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="StartTimeEdit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>45</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="EventNameLabel">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>8</y>
     <width>71</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code SemiBold</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>&lt;s&gt;Event&lt;/s&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="EndTimeEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>45</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <zorder>EndTimeEdit</zorder>
  <zorder>EndTime</zorder>
  <zorder>StartTimeEdit</zorder>
  <zorder>EventName</zorder>
  <zorder>StartTime</zorder>
  <zorder>DetailedInfomation</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>textBrowser</zorder>
  <zorder>EventNameLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
